pages/productAppExtension/[productId]/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { H4, Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo, useProductList } from "@lib/hooks";
+
+const ProductAppExtension = () => {
+    const router = useRouter();
+    const productId = Number(router.query?.productId);
+    const { error, isLoading, list = [] } = useProductList();
+    const { isLoading: isInfoLoading, product } = useProductInfo(productId, list);
+    const { description, is_visible: isVisible, name, price, type } = product ?? {};   
+    const typeCapitalized = type?.replace(/^\w/, (c: string) => c.toUpperCase());
+
+    if (isLoading || isInfoLoading) return <Loading />;
+    if (error) return <ErrorMessage error={error} />;
+
+    return (
+        <>
+            <Panel header="Basic Information" margin="xxSmall">
+                <H4 margin="none">Product name</H4>
+                <Text>{name}</Text>
+                <H4 margin="none">Product type</H4>
+                <Text>{typeCapitalized}</Text>
+                <H4 margin="none">Default price (excluding tax)</H4>
+                <Text>${price}</Text>
+                <Text>{isVisible}</Text>
```

Comment:
`isVisible` doesn't seem to be showing - let's also add a header for this

```suggestion
                <H4 margin="none">Visible on storefront</H4>
                <Text>{isVisible}</Text>
```